keyboards/handwired/polykybd/lang/_patch_named_glyphs.py

```
@@ -0,0 +1,55 @@
+    _r+=1
+_wb.close()
+START=_r                                     # first blank row after last filled glyph
+
+def rowxml(n, name, code):
+    return (f'<row r="{n}" customFormat="false" ht="12.8" hidden="false" '
+            f'customHeight="false" outlineLevel="0" collapsed="false">'
+            f'<c r="A{n}" t="inlineStr"><is><t xml:space="preserve">{escape(name)}</t></is></c>'
+            f'<c r="B{n}" t="inlineStr"><is><t xml:space="preserve">{escape(code)}</t></is></c>'
+            f'</row>')
+
+tmp="/tmp/_ng_work"; shutil.rmtree(tmp,ignore_errors=True); os.makedirs(tmp)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/_ng_work"

(S108)